src/client/components/localisation/view_model.ts

```diff
@@ -52,6 +55,13 @@ export class LocalisationViewModel {
   }
 
   @computed
+  private get balls(): Object3D[] {
+    return this.model.robots
+      .filter(robotModel => robotModel.visible)
```

Comment:
I think the grouping should be LocalisationRobotModels, since the ball and robot are tightly coupled, wouldn't ever make sense to show balls without the attached robots.